### PR TITLE
feat: report package link

### PR DESCRIPTION
**Pull request by gabewomble**
Fixes https://github.com/cdklabs/construct-hub-internal/issues/1

Also adds package version to the Package Page header

Screen Shot 2021-07-21 at 1 50 25 PM
Screen Shot 2021-07-21 at 1 51 56 PM

### CODE REVIEW

File: src/views/Package/components/OperatorArea/OperatorArea.tsx [hunk 1]
```diff
@@ -74,6 +81,12 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
       items.push(<>Registry: {registryLink}</>);
     }
 
+    items.push(
```
Comment (iliapolo) [inline src/views/Package/components/OperatorArea/OperatorArea.tsx:84]: The location of the link feels out of place to me. It's different than other links we show here and I think deserves a place of its own. This also makes it hard to notice it. 

Can we move to the top right corner of the panel?

File: src/views/Package/components/OperatorArea/OperatorArea.tsx [hunk 2]
```diff
@@ -74,6 +81,12 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
...
+      <Link color="blue.500" href="mailto:abuse@amazon.com">
```
Comment (iliapolo) [inline src/views/Package/components/OperatorArea/OperatorArea.tsx:85]: Maybe prefill the subject? Will make it easy for us to filter those out later.  

```suggestion
      <Link color="blue.500" href="mailto:abuse@amazon.com?subject={`ConstructHub - Report of abusive package: ${package}`}">
```

**Review by iliapolo — CHANGES_REQUESTED** [review]
Marking as changes requested to better filter out PR's that still need a review.

File: src/views/Package/components/OperatorArea/OperatorArea.tsx (hunk 1 above)
Comment (gabewomble) [reply src/views/Package/components/OperatorArea/OperatorArea.tsx:84]: I can move it to the top, but placing it top right seems out of place

**Review by iliapolo — APPROVED** [review]

**Pull request merged**